Makefile

```diff
@@ -5,7 +5,7 @@ IMG ?= fdb-kubernetes-operator:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true,maxDescLen=0"
 
-CONTROLLER_GEN_VERSION ?= 0.2.4
+CONTROLLER_GEN_VERSION ?= 0.5.0
```

Comment:
Seems like I came across this issue https://github.com/kubernetes-sigs/controller-tools/issues/529 🤔 I take a look if we can workaround this before upgrading the spec to apiextenstions v1

Reply:
I wonder if there's a way to leave the core types that we're embedding in our CRD out of the CRD definition. Most of the content in our CRD is for types that we don't control and don't have an authoritative definition for, since they depend on the server's Kubernetes version.

Reply:
As long as we want to have the possibility to override more or less all settings I don't see a way.